requirements/base.in

```diff
@@ -24,3 +24,4 @@ python-magic
 rules
 sentry-sdk[django]
 shapely
+soil_id @ https://github.com/techmatters/soil-id-algorithm/archive/ddf718a.zip
```

Comment:
question: does this now work with the more idiomatic syntax
```
git@https://github.com/techmatters/soil-id-algorithm@ddf718a
```

Reply:
That generates a syntax error.  If we try 
```
soil_id @ git+https://github.com/techmatters/soil-id-algorithm
```
then `make lock` runs okay, but make install will fail
```
ERROR: Can't verify hashes for these requirements because we don't have a way to hash version control repositories:
```

We need to use a zip so we can get a hash for it.

Reply:
ah i see, it's a `pip-tools` limitation, okie